nixos/roles/docker.nix

```diff
@@ -23,6 +23,14 @@ in
       flyingcircus.users.serviceUsers.extraGroups = [ "docker" ];
       virtualisation.docker.enable = true;
 
+      boot.kernel.sysctl = {
+        # IPv4 forwarding is enabled by default in the NixOS docker
+        # module, but as the platform overrides the upstream settings
+        # we need to in turn override the platform defaults here.
+        "net.ipv4.conf.all.forwarding" = fclib.mkOverridePlatformModule true;
```

Comment:
This stacking of overrides design is a bit unfortunate, but definitely good enough for a hotfix. I also do not immediately see how we can do better, so we might be stuck with that design even after reconsiderations later on.